client/src/components/BackupCodes/Verify.js

```diff
@@ -86,13 +86,24 @@ class Verify extends Component {
    */
   renderDescription() {
     const { ss: { i18n } } = window;
+    const { method } = this.props;
 
     return (
       <p>
         {i18n._t(
           'MFABackupCodesVerify.DESCRIPTION',
           'Use one of the recovery codes you received'
         )}
+        &nbsp;
+        {method && method.supportLink &&
+          <a
+            href={method.supportLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {i18n._t('MFARegister.RECOVERY_HELP', 'How to use recovery codes.')}
```

Comment:
This is different - is that by design? It's in its own component so it's fine if you want to leave it there, just asking

Reply:
Yeah, it's by design :)